lambdas/vonage_api_ingestion/main.py

```diff
@@ -234,122 +220,94 @@ def list_subfolders_in_directory(s3_client,bucket,directory):
+            importstring = path_dictionary["Prefix"]
 
-        importstring = re.search("[0-9]{4}-[0-9]{2}-[0-9]{2}", importstring).group()
-        # print(f'Date with /: {importstring}')
+            importstring = re.search("[0-9]{4}-[0-9]{2}-[0-9]{2}", importstring).group()
 
-        # print(f'Cleaning "{importstring}"')
+            list_of_raw_dates.append(importstring)
 
-        # print(f'Output "{importstring}"')
-        list_of_raw_dates.append(importstring)
+        list_of_raw_dates = sorted(list_of_raw_dates, key=lambda date: datetime.strptime(date, "%Y-%m-%d"),
+                                   reverse=True)
 
-      list_of_raw_dates = sorted(list_of_raw_dates, key=lambda date: datetime.strptime(date, "%Y-%m-%d"), reverse=True)
+        largest_value = list_of_raw_dates[0]
+        return largest_value
 
-      # print(list_of_raw_dates)
-      largest_value = list_of_raw_dates[0]
-      return largest_value
 
 def get_latest_file(list_of_import_years: list) -> str:
-  list_of_raw_dates = []
-  # print(list_of_raw_dates)
+    list_of_raw_dates = []
 
-  # print(f'Cleaning Import Years')
-  if len(list_of_import_years) == 0:
-      return None
-  else:
-      for subfolder in list_of_import_years:
-        #print(type(subfolder))
-        path_dictionary = dict(subfolder)
-        #print(path_dictionary)
+    if len(list_of_import_years) == 0:
+        return None
+    else:
+        for subfolder in list_of_import_years:
+            path_dictionary = dict(subfolder)
 
-        importstring = path_dictionary["Key"]
-        # print(f'Raw Prefix: {importstring}')
+            importstring = path_dictionary["Key"]
 
-        # Cleaning remove the importdate
-        importstring = re.sub(string=importstring,
-                              pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}\/",
-                              repl="")
+            importstring = re.sub(string=importstring,
+                                  pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}\/",
+                                  repl="")
 
-        importstring = re.search("[0-9]{4}-[0-9]{2}-[0-9]{2}", importstring).group()
-        # print(f'Date with /: {importstring}')
+            importstring = re.search("[0-9]{4}-[0-9]{2}-[0-9]{2}", importstring).group()
 
-        # print(f'Cleaning "{importstring}"')
+            list_of_raw_dates.append(importstring)
 
-        # print(f'Output "{importstring}"')
-        list_of_raw_dates.append(importstring)
+        list_of_raw_dates = sorted(list_of_raw_dates, key=lambda date: datetime.strptime(date, "%Y-%m-%d"),
+                                   reverse=True)
 
-      list_of_raw_dates = sorted(list_of_raw_dates, key=lambda date: datetime.strptime(date, "%Y-%m-%d"), reverse=True)
+        largest_value = list_of_raw_dates[0]
+        return largest_value
 
-      # print(list_of_raw_dates)
-      largest_value = list_of_raw_dates[0]
-      return largest_value
 
 # Takes a list of "importyear=X" values and finds the largest one
```

Comment:
consider making this a docstring for the function rather than a comment 
https://peps.python.org/pep-0257/#what-is-a-docstring